maintainers/maintainer-list.nix

```diff
@@ -4682,6 +4682,12 @@
     githubId = 1447245;
     name = "Robin Gloster";
   };
+  gnd = {
```

Comment:
This should be done in a separate commit.

Reply:
Fixed!